[\#2176 Issue](https://github.com/rear/rear/issues/2176) `closed`: mount cifs problem - but mkbackuponly is working
===================================================================================================================

**Labels**: `support / question`, `fixed / solved / done`,
`won't fix / can't fix / obsolete`

#### <img src="https://avatars.githubusercontent.com/u/594654?v=4" width="50">[maul75](https://github.com/maul75) opened issue at [2019-07-06 08:40](https://github.com/rear/rear/issues/2176):

All is working if I make only a "rear mkbackuponly" - as soon as I try
to make the rescue as well, it is not working "rear mkbackup or rear
mkrescue" - what I can see the output like:

`ERROR: Mount command 'mount -v -o rw,guest //myshare/backup/rear /tmp/rear.JjLlFk2ydwPH80s/outputfs' failed.`

It looks as if the mount is done by guest? But the mkbackuponly does
correctly move the file to the cifs share? Hope someone could give me a
tip what is going wrong?

-   ReaR version ("/usr/sbin/rear -V"):  
    Relax-and-Recover 2.4

-   OS version ("cat /etc/rear/os.conf" or "lsb\_release -a" or "cat
    /etc/os-release"):  
    CentOS Linux release 7.6.1810 (Core)

-   ReaR configuration files ("cat /etc/rear/site.conf" and/or "cat
    /etc/rear/local.conf"):

<!-- -->

    OUTPUT=ISO
    OUTPUT_URL="cifs://myshare/backup/rear"
    BACKUP=NETFS
    BACKUP_URL="cifs://myshare/backup/rear"
    BACKUP_PROG_EXCLUDE=("${BACKUP_PROG_EXCLUDE[@]}" '/dev/*' '/mnt/*' '/media/*' '/proc/*' '/sys/*' '/tmp/*' '/var/tmp/*' '/var/crash/*')
    BACKUP_OPTIONS="credentials=/etc/rear/cifs"

------------------------------------------------------------------------

\[Export of Github issue for
[rear/rear](https://github.com/rear/rear).\]
